mantle/cmd/ore/aws/upload.go

```diff
@@ -154,24 +152,32 @@ func runUpload(cmd *cobra.Command, args []string) error {
 		fmt.Fprintf(os.Stderr, "unknown AMI name; specify --ami-name\n")
 		os.Exit(2)
 	}
-
-	if uploadDiskSizeInspect {
-		imageInfo, err := util.GetImageInfo(uploadFile)
-		if err != nil {
-			fmt.Fprintf(os.Stderr, "Unable to query size of disk: %v\n", err)
-			os.Exit(1)
+	if uploadCreateWinLIAMI {
+		if uploadWinLIwindowsServerAMI == "" {
+			fmt.Fprintf(os.Stderr, "--windows-ami must be provided with --winli\n")
+			os.Exit(2)
 		}
-		plog.Debugf("Image size: %v\n", imageInfo.VirtualSize)
-		const GiB = 1024 * 1024 * 1024
-		uploadDiskSizeGiB = uint(imageInfo.VirtualSize / GiB)
-		// Round up if there's leftover
-		if imageInfo.VirtualSize%GiB > 0 {
-			uploadDiskSizeGiB += 1
+		if uploadWinLIwindowsServerAMI == "" {
```

Comment:
typo:

```suggestion
		if uploadWinLIInstanceType == "" {
```

Though I am wondering if we should hardcode a default and not 100% require it.